src/js/components/Widgets/ShowMoreButtons.jsx

```diff
@@ -5,12 +5,52 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import styled from 'styled-components';
 import { renderLog } from '../../common/utils/logging';
-
+import lookupPageNameAndPageTypeDict from '../../utils/lookupPageNameAndPageTypeDict';
+import TagManager from 'react-gtm-module';
+import VoterStore from '../../stores/VoterStore';
 
 class ShowMoreButtons extends React.Component {
-  render () {
-    renderLog('ShowMoreButtons');  // Set LOG_RENDER_EVENTS to log all renders
-    const { classes, showLessCustomText, showMoreButtonsLink, showMoreButtonWasClicked, showMoreCustomText, showMoreId } = this.props;
+  
+  constructor(props) {
+    super(props);
+    this.handleShowMoreClick = this.handleShowMoreClick.bind(this);
+    this.pushDataLayer = this.pushDataLayer.bind(this);
+  }
+
+  handleShowMoreClick = () => {
+    const { showMoreId, showMoreButtonWasClicked, trackInGTM, showMoreButtonsLink } = this.props;
+    const { location: { pathname: currentPathname } } = window; // Get path here
+    //if (trackInGTM) {
```

Comment:
I think having a trackInGTM variable makes sense. If you comment an if/else block out like this, please adjust the indenting of lines 25 & 26 to avoid the ESLINT warnings. On that note, make sure to run `npm run lint` before pull requests so you can fix ESLINT warnings.